decred/tests/unit/dcr/test_txscript.py

```diff
@@ -2845,14 +2888,14 @@ def pkAddr(b):
         ]
 
         def checkAddrs(a, b, name):
-            assert len(a) == len(b), (
-                f"extracted address length mismatch. expected {len(a)}, got {len(b)} for test {name}"
-            )
+            assert len(a) == len(
+                b
+            ), f"extracted address length mismatch. expected {len(a)}, got {len(b)} for test {name}"
 
             for av, bv in zip(a, b):
-                assert av.scriptAddress() == bv.scriptAddress(), (
-                    f"scriptAddress mismatch. expected {av.scriptAddress().hex()}, got {bv.scriptAddress().hex()} for test {name}"
-                )
+                assert (
+                    av.scriptAddress() == bv.scriptAddress()
+                ), f"scriptAddress mismatch. expected {av.scriptAddress().hex()}, got {bv.scriptAddress().hex()} for test {name}"
```

Comment:
Still a too long line, I'm afraid. You could either split it and rely on implicit string concatenation, or explicitly calling `format`:

```python
"scriptAddress mismatch. expected {}, got {} for test {name}".format(
    av.scriptAddress().hex(), bv.scriptAddress().hex())
```

Anyway we can deal with this later, approving for now.